src/pages/Ranking/index.js

```diff
@@ -14,73 +15,108 @@ import './styles.css';
-                                  className="daily-data-word"
-                                  style={{ backgroundColor: results[index] }}
-                                >
-                                  {letter}
-                                </span>
-                              ))}
-                            </div>
-                          ))}
-                        </>
-                      )}
-                    </div>
-                  )}
-                </ListRow>
-              );
-            }
-          )}
-        </TabPanel>
-        <TabPanel>
-          <h2>In Progress...</h2>
-        </TabPanel>
-      </Tabs>
+                return (
+                  <ListRow
+                    key={email}
+                    classProps={{ isCurrentUser, lost, isDisabled }}
+                    onClick={() => setExpandedUser(isExpanded ? '' : email)}
+                    disabled={!currentUserPlayed}
+                    name={name}
+                    photo={photo}
+                    leftText={position}
+                    rightText={lost ? 'X' : attempts}
+                    icon={isExpanded ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+                    showIcon={currentUserPlayed}
+                  >
+                    {currentUserPlayed && (
+                      <div
+                        className={cn('daily-data-words', { isExpanded })}
+                        style={{
+                          height: isExpanded ? attemptedWords.length * WORD_HEIGHT : 0,
+                        }}
+                      >
+                        {isExpanded && (
+                          <>
+                            {attemptedWords.map(({ word = '', results }) => (
+                              <div key={word} className="daily-data-word-container">
+                                {word.split('').map((letter, index) => (
+                                  <span
+                                    key={`${word}-${index}`}
+                                    className="daily-data-word"
+                                    style={{ backgroundColor: results[index] }}
+                                  >
+                                    {letter}
+                                  </span>
+                                ))}
+                              </div>
+                            ))}
+                          </>
+                        )}
+                      </div>
+                    )}
+                  </ListRow>
+                );
+              }
+            )}
+          </TabPanel>
+          <TabPanel>
+            {currentStreakRankingData.map(
+              ({ currentStreak, lastDatePlayed, position, user: { email, name, photo, uid } }) => {
+                const isCurrentUser = email === currentUser;
+                return (
+                  <ListRow
+                    key={`${email}-${currentStreak}-current-streak`}
```

Comment:
Isn't just the `email` unique?

Reply:
Yes but I though maybe it could give conflicts with the other tab ListRow so I added more things just in case 😂 

![image](https://user-images.githubusercontent.com/8755889/209727305-20f5d604-fc7d-45d5-a187-dfbab8c27c6a.png)

Reply:
I tried and it works fine so I can just leave the email yes, thanks!